docs/design/flow-traffic-mirroring/delete-mirror-point.puml

```diff
@@ -0,0 +1,52 @@
+@startuml
+title Delete flow traffic mirror point
+
+boundary Northbound
+box "Flow HS Topology" #LightGreen
+    participant FlowDeleteMirrorPointHub
+    participant FlowResourcesManager
+    participant CommandBuilder
+    participant SpeakerWorker
+end box
+database OrientDB
+
+Northbound -> FlowDeleteMirrorPointHub : Delete flow mirror point
+activate FlowDeleteMirrorPointHub
+ ==Validation==
+FlowDeleteMirrorPointHub -> OrientDB : Get flow
+FlowDeleteMirrorPointHub <- OrientDB : Requested flow
+alt The flow mirror point exists
+    Northbound <- FlowDeleteMirrorPointHub : Flow mirror point details
+else The flow mirror point not found
+    Northbound <- FlowDeleteMirrorPointHub : Flow mirror point deletion error
+end
+
+ == Flow mirror point rules deinstallation ==
```

Comment:
"deinstallation"... a little weird term. Maybe we can use `delete` (sane as in flowhs docs) or at least `uninstall`?

Reply:
Fixed.